### PR TITLE
Add link to ApacheCon per Branding; fix a few other branding standards issues; fix sitemap.xml refs to localhost

### PR DESCRIPTION
**Pull request by srowen**

### CODE REVIEW

**Comment by srowen** [comment]
Here's how the new events link looks at the moment: 
screen shot 2018-04-09 at 11 46 01

File: _layouts/global.html [hunk 1]
```diff
@@ -211,10 +216,12 @@ <h5>Latest News</h5>
 
 <footer class="small">
   <hr>
-  Apache Spark, Spark, Apache, and the Spark logo are <a href="{{site.baseurl}}/trademarks.html">trademarks</a> of
-  <a href="https://www.apache.org">The Apache Software Foundation</a>.
+  Apache Spark, Spark, Apache, the Apache feather logo, and the Apache Spark project logo are either registered
```
Comment (srowen) [inline _layouts/global.html:219]: This new text is pulled from the trademarks page and should make https://whimsy.apache.org/site/project/spark happier.

File: index.md [hunk 2]
```diff
@@ -181,9 +181,3 @@ navigation:
     </ul>
   </div>
 </div>
-
-<div class="row">
```
Comment (srowen) [inline index.md:185]: This is a redundant download link at the bottom; not strictly related, but, I removed it as it's already in the right nav.

**Comment by rxin** [comment]
LGTM

**Comment by mateiz** [comment]
@srowen this is okay, but I would've put it below the Download Spark button since it will be a permanent fixture on the page. What do you think?

**Comment by srowen** [comment]
My thinking was that it went more naturally with the block of news, which frequently announces events. If you have a moderate preference for the other way I can just change it.

**Pull request merged**